frontend/src/main/web/src/components/Heading.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react' // eslint-disable-line
+import { flattenThemeClasses } from '../utils/styleUtils'
+
+const Heading = ({
+  children,
+  level,
+  theme,
+  ...props
+}) => {
+  const classes = {
+    base: {
+      c: 'C(pri)',
+      m: 'M(0)',
+      fz: 'Fz(ms0)',
+      fw: 'Fw(i)',
+      lh: 'Lh(1)'
+    }
+  }
+  const headingClasses = flattenThemeClasses(classes, theme)
+  return (<h1 {...props} className={headingClasses} >
```

Comment:
Wrap the tag to the next line or remove the wrapping parens. Parens are only needed around JSX if the first tag will start on the next line.